src/test/Testcafe/TestData.js

```diff
@@ -10,31 +10,23 @@ let environment = (process.env.ENV).toLowerCase();
 //Role for an user who has rights to deploy an app to a test environment
 export const AutoTestUser = Role(app.baseUrl, async t => {
   var userName = config[environment].autoTestUser;
-
-  //Get password from environments
+  //Get password from environment variable
```

Comment:
this comment doesn't really add any value, can deduce it from the code itself. Explain why not how